lib/gtag.js

```diff
@@ -0,0 +1,29 @@
+export const GA_TRACKING_ID = process.env.GA_TRACKING_ID;
```

Comment:
Can you make this file to typescript?

Reply:
Also lib does not seem to be correct folder for this


Can you move this to the `shared` folder?

ie., `/shared/gtag.ts`